benchmarks/OpOptimization/BatchMatmul/TVM/CPU/main.py

```diff
@@ -0,0 +1,127 @@
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+# ===---------------------------------------------------------------------------
+#
+# This file implements the entry for AutoScheduler BatchMatMul on CPU.
+# Autoscheduler is TVM's next-generation performance tuning tool,
+# which can automatically generate search spaces for optimizing tensor expressions.
+# TVM is an Apache-2.0 licensed project.
+# See the TVM license at: https://github.com/apache/tvm/blob/main/LICENSE
+#
+# ===---------------------------------------------------------------------------
+from batch_matmul_cpu import *
+import numpy
+import timeit
+import tvm.testing
+import mxnet as mx
```

Comment:
Please remove unnecessary imports. This package is causing Segfault on my machine as well.